test/OutsideClickHandler_test.jsx

```diff
@@ -1,12 +1,14 @@
 import React from 'react';
-import { expect } from 'chai';
+import { use, expect } from 'chai';
 import sinon from 'sinon-sandbox';
 import { shallow, mount } from 'enzyme';
 import wrap from 'mocha-wrap';
 import contains from 'document.contains';
-
+import chaiEnzyme from 'chai-enzyme';
 import OutsideClickHandler from '../src/OutsideClickHandler';
 
+use(chaiEnzyme());
```

Comment:
not so sure this complexity pays for itself in these two tests

Reply:
Yes, I hesitated, but it was either have a complex way of writing the test at the cost of readability, or adding complexity through import. I opted for the second option, in the hope that future tests could benefit from chai-enzyme.

Reply:
I ended up removing this; it's a one-liner with basic chai, so there's zero readability hit.